src/ViewHelpers/LayoutViewHelper.php

```diff
@@ -11,6 +11,7 @@
 use TYPO3Fluid\Fluid\Core\ViewHelper\AbstractViewHelper;
 use TYPO3Fluid\Fluid\Core\ViewHelper\PostParseInterface;
```

Comment:
I think we've got this and a few other ones still lingering around. Will follow up with a final cleaning after my local development stuff is all pushed (about 50% is opened as PRs the rest is still waiting here).